ansible_collections/arista/avd/docs/input-variable-validation-BETA.md

```diff
@@ -56,34 +60,36 @@ tenants:
 
 The input data will be converted automatically and validated according to the new model to allow for a gracious introduction of this data-model changes in late 3.x versions.
 
-Type conversion is turned on by default but can be disabled or adjusted with the options described in the next section.
+Type conversion is turned on by default.
+
+!!! CAUTION
```

Comment:
I am wondering if this is wise to even allow it given the drawbacks :) ..

Reply:
We have a few special use cases where disabling could make sense, but we keep it hidden?

Reply:
we can revisit if we see problems